benchmark/benchmark.go

```diff
@@ -193,6 +122,27 @@ func DoUnaryCall(tc testpb.TestServiceClient, reqSize, respSize int) {
 	}
 }
 
+//DoStreamingcall performs a streaming RPC with given stub and request and response size.client side
+func DoStreamingCall(tc testpb.TestServiceClient, reqSize, respSize int) {
+	pl := newPayload(testpb.PayloadType_COMPRESSABLE, reqSize)
+	req := &testpb.SimpleRequest{
+		ResponseType: pl.Type,
+		ResponseSize: int32(respSize),
+		Payload:      pl,
+	}
+	stream, err := tc.StreamingCall(context.Background())
+	if err != nil {
+		grpclog.Fatalf("TestService/Streaming call rpc failred: ", err)
```

Comment:
change the log to something like:

grpclog.Fatalf("%v.StreamingCall(_) = _, %v", tc, err)